From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Colin Cross <ccross@android.com>
Date: Wed, 15 Feb 2023 15:02:51 -0800
Subject: [PATCH 2/7] Add VanillaIceCream to aapt2 codenames

Bug: 279492191
Test: m checkbuild
Change-Id: Ia8bd807ab25260ff6716e2aea36e4d3e8567eb22
Merged-In: Ia8bd807ab25260ff6716e2aea36e4d3e8567eb22
---
 tools/aapt2/SdkConstants.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/aapt2/SdkConstants.cpp b/tools/aapt2/SdkConstants.cpp
index 0aca2939cee0..59982998b24f 100644
--- a/tools/aapt2/SdkConstants.cpp
+++ b/tools/aapt2/SdkConstants.cpp
@@ -26,8 +26,8 @@ using android::StringPiece;
 namespace aapt {
 
 static ApiVersion sDevelopmentSdkLevel = 10000;
-static const auto sDevelopmentSdkCodeNames =
-    std::unordered_set<StringPiece>({"Q", "R", "S", "Sv2", "Tiramisu", "UpsideDownCake"});
+static const auto sDevelopmentSdkCodeNames = std::unordered_set<StringPiece>(
+    {"Q", "R", "S", "Sv2", "Tiramisu", "UpsideDownCake", "VanillaIceCream"});
 
 static const std::vector<std::pair<uint16_t, ApiVersion>> sAttrIdMap = {
     {0x021c, 1},
